tests/Integration/Behaviour/Features/Context/Domain/CustomerGroupFeatureContext.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain;
+
+use Behat\Gherkin\Node\TableNode;
+use Exception;
+use PHPUnit\Framework\Assert;
+use PrestaShop\Decimal\DecimalNumber;
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command\AddCustomerGroupCommand;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\Query\GetCustomerGroupForEditing;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\QueryResult\EditableCustomerGroup;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupId;
+use Tests\Integration\Behaviour\Features\Context\CommonFeatureContext;
+
+class CustomerGroupFeatureContext extends AbstractDomainFeatureContext
+{
+    /**
+     * @param string $customerGroupReference
+     *
+     * @return EditableCustomerGroup
+     */
+    private function getCustomerGroupForEditing(string $customerGroupReference)
```

Comment:
return type

Reply:
+ check methods ordering by scope.
public methods goes at the top,
then protected,
then private